apps/studio/src/lib/db/clients/index.ts

```diff
@@ -1,12 +1,21 @@
 // Copyright (c) 2015 The SQLECTRON Team
 
-import mysql from './mysql';
-import postgresql from './postgresql';
-import sqlserver from './sqlserver';
-import sqlite from './sqlite';
+import mysql, { knex as mysqlKnex } from './mysql';
+import postgresql, { knex as postgresKnex } from './postgresql';
+import sqlserver, { knex as sqlServerKnex } from './sqlserver';
+import sqlite, { knex as sqliteKnex } from './sqlite';
 import cassandra from './cassandra';
 import bigquery from './bigquery.js';
+import { Dialect } from '@shared/lib/dialects/models';
+import { Knex } from 'knex';
 
+export function knexOf(dialect: Dialect): Knex | undefined {
```

Comment:
I don't like these types of centralized switches on database type as they're easy to miss when adding a new database, but I think this will go away if we keep arrays as JSON arrays when in the UI.